jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Attachment.java

```diff
@@ -343,6 +343,12 @@ String loadAgentLibrary(String agentLibrary, String options,
 				return Response.EXCEPTION_AGENT_INITIALIZATION_EXCEPTION
 						+ status;
 			}
+		} else {
+			try {
+				saveLocalConnectorAddress();
+			} catch (IbmAttachOperationFailedException e) {
+				return Response.EXCEPTION_AGENT_INITIALIZATION_EXCEPTION + "loadAgentLibrary: " + e.getMessage();
```

Comment:
Please add a space between EXCEPTION_AGENT_INITIALIZATION_EXCEPTION and "loadAgentLibrary: "